app/javascript/packages/verify-flow/steps/personal-key-confirm/personal-key-confirm-step.tsx

```diff
@@ -18,6 +19,17 @@ function PersonalKeyConfirmStep(stepProps: PersonalKeyConfirmStepProps) {
         <PersonalKeyStep {...stepProps} />
       </FormStepsContext.Provider>
       <Modal>
+        <div className="pin-top pin-x">
+          <div className="display-flex flex-column flex-align-center">
+            <img
+              className="top-neg-3"
+              alt=""
+              height="60"
+              width="60"
+              src={getAssetPath('p-key.svg')}
+            />
+          </div>
+        </div>
```

Comment:
When I tested this, it looks like the icon is within the content, when I'd expect it should be half-outside.

![image](https://user-images.githubusercontent.com/1779930/164091020-c3414e63-bd59-4cce-818b-c6df859f5c22.png)

One option could be to add a `position-relative` class to the image, so that the `top-neg-3` takes effect. Alternatively, we might be able to do a bit of flattening and class consolidation by adding everything to the top-level `div`:

```suggestion
        <div className="pin-top pin-x top-neg-3 display-flex flex-column flex-align-center">
          <img alt="" height="60" width="60" src={getAssetPath('p-key.svg')} />
        </div>
```

Reply:
For some reason I thought position absolute didn't work with flex but it seems I was wrong! Flattening it def makes more sense then, no reason for an extra div.